### ISSUE TITLE
Incorrect gun sound assets.

### ISSUE THREAD

**Issue by sergeantpwnr** [GaijinEntertainment/fmod_studio_warthunder_for_modders#300]
Hello. I have noticed that several of the gun sound assets do not correspond to the ones in the game.

Most recently overhaul gun sounds like the Gsh-30-1 and the BK27 do not contain their updates from Seek & Destroy.

The GAU-8 sound which was updated a couple months ago also seems to only partially have been updated in the project.

**Comment by Uchuujin01** [comment]
Seek and Destroy gun sound changes were rolled back to their pre-update state, for gau-8 in particular. Some sounds may have not updated, but anyway you are free to make your own sounds, do you really need updated vanilla versions?

**Comment by sergeantpwnr** [comment]
Yes. My intention is to overhaul the flare sound and leave everything else the same. Y'all did good on the gun sounds! 

Am I able to download the new sounds from the pre-revert branch? Also, the GAU-8 only seems to be missing the sound of the shells, which leads me to believe it's an error with the bank.
